.github/workflows/build-master.yml

```diff
@@ -0,0 +1,26 @@
+name: Build Master Branch
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 11
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+      - name: Grant execute permission for gradlew
+        run: chmod +x gradlew
+      - name: Build with Gradle
+        env:
+          packageUser: ${{ secrets.BALLERINA_BOT_USERNAME }}
+          packagePAT: ${{ secrets.BALLERINA_BOT_TOKEN }}
+          packagePublishRepo: ${{ github.repository }}
+        run: |
+          ./gradlew clean build publish --stacktrace --scan --console=plain --no-daemon
```

Comment:
If you want, you can add `Archive Error Log` step to archive the `ballerina-internal.log` incase of a failure. You can find an example here:
https://github.com/ballerina-platform/module-ballerina-graphql/blob/89b2867b7a06231f24613af20cf9af1221236050/.github/workflows/build-master.yml#L29

Reply:
This seems like a good addition. I will add this as well